react-search-component/my-app/src/FilteredItems.js

```diff
@@ -0,0 +1,24 @@
+export default function FilteredItems(props) {
+  //Lowercase each string
+  const lowerCaseList = props.list.map((element) => element.toLowerCase());
+  const lowerCaseInputString = props.string.toLowerCase();
+
+  //Create array of filtered <li> element with each string
+  const filteredList = [];
+
+  console.log(props.string);
+
+  for (let i = 0; i < props.list.length; i++) {
+    if (lowerCaseList[i].includes(lowerCaseInputString)) {
+      let item = <li key={props.list[i]}>{props.list[i]}</li>;
```

Comment:
Use `const` whenever you can. You should first declare all variables `const` and then change them to `let` only if you have to.